build.gradle

```diff
@@ -219,6 +222,15 @@ subprojects {
         classifier "test"
     }
 
+    task noInternalJavadoc(type: Javadoc) {
```

Comment:
Please have it in a separate `.gradle` file, so it is easier to reuse in future.